vnext/Microsoft.ReactNative/Views/cppwinrt/ViewPanel.idl

```diff
@@ -15,25 +15,23 @@ namespace Microsoft.ReactNative
 
     // Public Methods
     void InsertAt(UInt32 index, XAML_NAMESPACE.UIElement value);
-      void RemoveAt(UInt32 index);
-      void Clear();
+    void RemoveAt(UInt32 index);
+    void Clear();
 
     void FinalizeProperties();
-      XAML_NAMESPACE.Controls.Border GetOuterBorder();
+    XAML_NAMESPACE.Controls.Border GetOuterBorder();
 
     // Public Properties
     XAML_NAMESPACE.Media.Brush ViewBackground { get; set; };
     XAML_NAMESPACE.Thickness BorderThickness { get; set; };
     XAML_NAMESPACE.Media.Brush BorderBrush { get; set; };
     XAML_NAMESPACE.CornerRadius CornerRadius { get; set; };
-    Boolean ClipChildren { get; set; };
```

Comment:
these are all breaking changes in the public API surface (though we don't expect folks to use ViewPanel directly, they may be)

Reply:
Indeed, this change is "stacked" on a previous PR that is marked appropriately as a breaking change.